tasks/sass.js

```diff
@@ -30,11 +32,11 @@ export default async function () {
     const css = compileSassFile(file);
 
     // Create the output assets directory if it doesn't already exist
-    await mkdir(`${paths.output}/assets`, {
+    await mkdir(`${paths.outputAssets}`, {
       recursive: true,
     });
 
     // Write the CSS file
-    await writeFile(`${paths.output}/assets/${filename}`, css);
+    await writeFile(`${paths.outputAssets}/${filename}`, css);
```

Comment:
This feels like it might be an optimisation for a future PR, but it'd be great to add fingerprints to asset filenames so that we can cache them indefinitely.

Reply:
In my personal projects I've done this by appending a query string when referencing the file in Nunjucks. It has the benefit of being simpler to implement, as the name of the file itself doesn't change, but also means the fingerprint isn't tied to the file contents and will typically be something else (e.g. the last build time). 

If we're fine with that approach, I can add it quickly.

Reply:
@querkmachine It's likely worth adding Terser for JS and CSSNano and Autoprefixer for the CSS in future PRs as well.

Reply:
The downside of that is that the cache for all of the assets will be invalidated for all files every time we deploy, whereas fingerprinting based on file contents means we can preserve the cache for the files that haven't changed.

Given how many images there are on the site, it would be good to apply the same approach to them as well.

Given I don't expect we'll be deploying often it's not the end of the world, but my preference would be to fingerprint the files based on their contents if possible.

Happy for this to be done later, or for us to use the query string approach for now and revisit if there's time near the end of the cycle – just feels like we're double-handling a little bit if we do that.

Reply:
@querkmachine Given the asset build happens in the `beforeBuild` hook, I'm wondering if we have the space to hash the files in the hook to get a hash based on content, add it to the filename then pass the file names to Eleventy's data so the layout can point to the right files.

For the images, I think the Eleventy Img takes care of adding a hash to the filenames when it runs, so hopefully that bit is covered "for free" 🤞🏻

Reply:
I've added Terser, Autoprefixer and cssnano as I imagine they're all fairly simple (and presumably, non-controversial) additions. 

Will leave fingerprinting for now. A cursory glance seems like it might be a little bit involved, especially around how you pass the fingerprinted names into Eleventy's data cascade or straight into Nunjucks.